bowling.js

```diff
@@ -0,0 +1,51 @@
+function BowlingGame() {
+  this.rolls = [];
+}
+
+BowlingGame.prototype.isSpare = function(roll1, roll2) {
+  return roll1 + roll2 === 10;
+}
+BowlingGame.prototype.isStrike = function(roll){
+  return roll === 10;
+}
+
+BowlingGame.prototype.calculateSpareBonus = function(frameIndex){
+  return this.rolls[frameIndex + 2];
+}
+BowlingGame.prototype.calculateStrikeBonus = function(frameIndex){
```

Comment:
Functions/Methods should do one thing - This should only calculate the Strike bonus.

However, it is also checking what roll the person is at - refactor this out of the function.

Tip: A strike bonus calculation will always be the same, regardless of what roll it is.